### ISSUE TITLE
Absence of reactivity in the lane of anti Kappa antiserum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9589]
**Preferred term label:**
https://pubmed.ncbi.nlm.nih.gov/26853951/

-- determine best label and definition. (? CD19 þ K þ lymphocytes )
-- Also spliot chronic and recurrent diarrhea (1/1  in this publication)

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000261
- label: Absent circulating immunoglobulin kappa chain
- parents: [Abnormal immunoglobulin level (HP:0010701)](https://hpo.jax.org/app/browse/term/HP:0010701)
- pmid: PMID:26853951
- def: Nondetectable kappa chain in the blood circulation.
- comment: Human immunoglobulin molecules are built up by a pair of identical heavy chains, identifying the immunoglobulin class (namely Gamma, Alpha, Mu, Delta and Epsilon), and by a pair of identical light chains, Kappa or Lambda alternatively, character- izing the immunoglobulin type. In humans, under normal condi- tions, Kappa light chains represent approximately 2/3 of the light chains of total immunoglobulins, both circulating and lymphocyte surface bound.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9589
- synonyms:
